DigitalLearningSolutions.Web/Views/LearningPortal/RecommendedLearning/RecommendedLearning.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@{
+  Layout = "../SelfAssessments/_Layout";
+  ViewData["Title"] = "Recommended learning";

```

Comment:
Added a basic model just containing the whole self assessment record for now, same as the old page.

Reply:
Not sure if this is ready for re-review, but just quickly checked this out. Looks like the layout is not quite working correctly with this. The breadcrumbs try to lead to /RecommendedLearning/Current with the controller change, which causes an error page to spring up instead. The additional breadcrumbs from the previous page aren't carried forward either.

Reply:
Fixed the link with no controller specified and added the rest of the breadcrumbs to the new page.

![image](https://user-images.githubusercontent.com/59561751/145383008-e0d16cd9-f5c2-4e51-9d7e-73078510ff39.png)

Reply:
Just because I've copied the breadcrumbs from the old filtered results view, which they're set up the same as here in. I can fix it to include the name, but I'd prefer to resolve it properly in the next subtask where I'm actually making the page/view model etc. properly.

Reply:
Fair enough,